recipe/bld.bat

```diff
@@ -12,21 +12,27 @@ if errorlevel 1 exit 1
     threading=multi ^
     link=static,shared ^
     -j%CPU_COUNT% ^
-    --without-python
+    --without-python -d+2
```

Comment:
What’s this last bit for?

Reply:
this enable b2 debug output, but it turns out the build was okay, it's just that the install layout changed